dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/common/DefaultIdentifiableObjectManager.java

```diff
@@ -1253,6 +1253,9 @@ private <T extends IdentifiableObject> List<T> findByUser(
         schema.getPersistedProperty(BaseIdentifiableObject_.LAST_UPDATED_BY) != null;
 
     UserDetails currentUserDetails = UserDetails.fromUser(user);
+    if (currentUserDetails == null)
+      throw new IllegalArgumentException(
```

Comment:
Hm.. maybe we should throw an exception linked to authority that also causes a redirect to the login? @netroms any thoughts?

Reply:
well, the `user` param in this method doesn't necessarily have to be the logged in `user`. Not sure if that would be the correct thing to do here. It looks like this method can take any `user` (doesn't have to be logged in) and finds objects linked them.

Reply:
AFAIK any user in DHIS2 is a "logged in" user. There are only a few special APIs one may use as a "unidentified" user, like the login page. Lets see what MS has to say :)

Reply:
had a chat with @netroms, he's happy to just delete the method as well as it's not used. So I'll update the PR to just remove this method completely.